app/src/main/AndroidManifest.xml

```diff
@@ -250,11 +250,11 @@
 
     <activity
         android:name="com.asfoundation.wallet.verification.ui.credit_card.VerificationCreditCardActivity"
-        android:label="@string/verification_settings_unverified_title"
+        android:label=""
         android:windowSoftInputMode="stateHidden" />
     <activity
         android:name="com.asfoundation.wallet.verification.ui.paypal.VerificationPaypalActivity"
-        android:label="@string/verification_settings_unverified_title"
+        android:label=""
         android:windowSoftInputMode="stateHidden" />
```

Comment:
is necessary keep the tag?

Reply:
This are necessary to not show the default label on the toolbar